src/openforms/registrations/contrib/objects_api/models.py

```diff
@@ -90,6 +104,17 @@ class ObjectsAPIConfig(SingletonModel):
         validators=[validate_rsin],
         help_text=_("Default RSIN of organization, which creates the INFORMATIEOBJECT"),
     )
+    content_json = models.TextField(
+        _("Json field"),
+        validators=[
+            DjangoTemplateValidator(
+                backend="openforms.template.openforms_backend",
+            ),
+            JsonTemplateValidator(max_length=300),
```

Comment:
This validation on the length does not make too much sense here, since once the `json.loads` happens, there will be all the extra data from the form/vairables that will make it longer. The check should happen after you have rendered the template and before you are about to convert it to JSON. 

I discussed with Sergei that this should then raise a `django.core.exceptions.SuspiciousOperation` if the rendered template larger is than a configurable size. To determine a reasonable default, we could time how long it takes with inputs of increasing size (1mb, ...) and decide what seems appropriate